modules/browsiRtdProvider.js

```diff
@@ -283,36 +248,37 @@ export const browsiSubmodule = {
   /**
    * get data and send back to realTimeData module
    * @function
-   * @param {adUnit[]} adUnits
-   * @param {function} onDone
+   * @param {string[]} adUnitsCodes
    */
-  getData: sendDataToModule,
-  init: init
+  getTargetingData: sendDataToModule,
+  init: init,
 };
 
-function init(config, gdpr, usp) {
+function init() {
   return true;
 }
 
-export function beforeInit(config) {
+function beforeInit(config) {
   const confListener = config.getConfig(MODULE_NAME, ({realTimeData}) => {
```

Comment:
I think only RTD-core should be a config listener -- it should then call the init function for each sub-module.

Reply:
Yeah I agree with this.

Seems like bad practice to gather data from here, when the core module should gather and just pass the params associated with browsi.